app/assets/stylesheets/top-bar.scss

```diff
@@ -2,6 +2,11 @@
 @import 'variables';
 @import 'mixins';
 
+:target {
+  padding-top: var(--header-height);
+  margin-top: calc(-1 * var(--header-height));
+}
+
```

Comment:
I think this is not optimal solution because of two reasons:

1. It adds additional spacing above the footnote list.
2. User can make header not being sticky in /settings/ux and in that case that fix isn't needed at all. So we should better target it (I believe we have specific body class applied when header is sticky/notsticky).

---

Generally there's much better solution for that issue: 

```
body.default-navbar-config { // targeting sticky header only
  .article .body [id] { // targeting any element with [id] attribute within article content
    scroll-margin-top: var(--header-height); // magic :)
  }
}
```

`scroll-magin-top` is much better for handling issues like that :)